### ISSUE TITLE
initialise browsing with true random sample

### ISSUE THREAD

**Issue by dobkeratops** [bbernhard/imagemonkey-core#203]
In both modes - add-labels and annotate:

(i) for most users it might be enough (and more visually inviting) to present a truly random sample : then you are one click away from work - I speculate this would be the best default. It could still be enough to dodge the "frustrating tasks" problem, and it would also speed up the decision (by giving some suggestions upfront). 

There would be an opportunity to enhance this behind the scene (e.g. the image clustering utility to ensure maximum variety in the set of suggestions, and possibly sorting by number of labels / number of annotations - but I hope a truly plain random sample would be enough to start with

(ii) if you streamline the clicks from *add labels* to *annotate* (perhaps clicking a label takes you to annotating that label)  , you could make "add labels" the default view- streaminling your whole UI

**So this is how I imagine it working:**

start:
[1] user imiediately sees a selection of random images,  with a search box to refine that *if needed* (the search box starts out just saying "random"?)

[2] <scroll a little and click an image..>

[3] you see the "add labels" page, which also has a quite big button "Annotate.." (instead of goto dropdown) forget 'validate:'  annotate does validation, and you could have another direct way to 'downvote' labels?)

.. and clicking the label in the labels page fires up annotation mode for that label (discover this through tooltip Labels: car, truck,.."click here to annotate truck.."). 

you then have the back button allowing you to stay focussed on the image and add labels, annotate them immiediately. 

So basically once you know your way around , it's just *2 clicks* to annotate something, from a random view. 1 click+label entry to add a label. You also free  up space on the top - unify the modes,   'add labels, annotate, explore..' explore is just "search the data".  Only advanced users need the explore data debug, so that can be relegated to a right click menu or even hotkey (perhaps a little "?" or ... menu indicator in a small line below the image) . That function was definitely useful in debugging , it just doesn't need to be prominent in the UI workflow
